nixos/tests/k3s/multi-node.nix

```diff
@@ -112,7 +112,7 @@ import ../make-test-python.nix ({ pkgs, lib, ... }:
           enable = true;
           role = "agent";
           serverAddr = "https://192.168.1.3:6443";
-          extraFlags = lib.toString [
+          extraFlags = builtins.toString [
```

Comment:
I think `lib.concatStringsSep " "` is more explicit, even though they both seem to do the same thing in this case. What do you think?